util/packages.go

```diff
@@ -74,8 +74,13 @@ func getPackagePaths(allPaths []string) ([]string, error) {
 			if info.IsDir() {
```

Comment:
I would take this opportunity to make this check a bit stronger.

If the execution reaches this line, we know that we should be looking at a directory whose name is a version. So how about we not only check if `info.IsDir()` but also check that `dirs[1]` [is a semver](https://godoc.org/github.com/Masterminds/semver#StrictNewVersion)?  That way we handle not only the unexpected _file_ case but also the unexpected _folder_ case as well.